test/Examples_Tests/src/Signatures_Spec.enso

```diff
@@ -0,0 +1,71 @@
+from Standard.Base import all
+
+from Standard.Test import all
+
+from project.Enso_Bin_Utils import all
+
+add_specs suite_builder = suite_builder.group "Standard.Base Signature Checks" group_builder->
+    setup = Api_Setup.create 'Standard' 'Base' [ "--docs=api" ]
+
+    group_builder.specify "Standard.Base.Meta" <|
+        snapshot = setup.api_dir / "Standard" / "Base" / "Meta.md"
+        actual = File.read_text snapshot
+        expected = """
+            ## Enso Signatures 1.0
+            ## module Standard.Base.Meta
```

Comment:
The following is the _API snapshot_ of `Standard.Base.Meta`. It should be checked on the CI with every integration and differences should be reported as an error.
```bash
sbt:enso> runEngineDistribution --docs=api --in-project distribution/lib/Standard/Base/0.0.0-dev/
```
and then copy the content of 
```bash
enso$ cat distribution/lib/Standard/Base/0.0.0-dev/docs/api/Standard/Base/Meta.md
```
here.

Should the `Meta` API change, one should regenerated the documentation.

Reply:
We should have instructions on how to update expectations in some more visible place (documentation? even comment here would be enough). Otherwise, devs will need to dig through git history or ping the original author, none of which are particularly good.

Reply:
I'd prefer to keep the signatures in a separate file (in `data` folder).

But if this will be soon replaced with #10507 than it seems OK as a temporary solution anyway.

Reply:
The manipulation with a separate file will be easier. Done in 2f8d126b68 including instructions how to regenerate the file.